newswires/client/src/SearchSummary.tsx

```diff
@@ -68,50 +74,33 @@ const SummaryBadge = ({
 
 	const { config, handleEnterQuery, toggleSupplier } = useSearch();
 
-	const handleRemoveBadge = (label: SearchTermBadgeLabel, value: string) => {
-		const categoryCodes = config.query.categoryCode ?? [];
-		const categoryCodesExcl = config.query.categoryCodeExcl ?? [];
-
-		switch (label) {
-			case 'Search term':
+	const handleRemoveBadge = (key: keyof Query, value: string) => {
```

Comment:
using key rather than label allows us to more easily handle similar parameter types with the same switch clause; felt worthwhile, while I was adding new keys